Importer/Event/ImportEvents.php

```diff
@@ -8,4 +8,6 @@
     const POST_FIELD_INTERPRET = 'EVENT_POST_FIELD_INTERPRET';
 
     const PRE_BIND_DATA = 'EVENT_PRE_BIND_DATA';
+
+    const POST_VALIDATION = 'EVENT_POST_VALIDATION';
```

Comment:
the order of this file is a bit strange. it seems it _was_ alphabetical but now its random. perhaps we could make it into the order they happen?